js/google-shopping.js

```diff
@@ -1,3 +1,93 @@
 // Write your solutions below
 
-console.log(products);
+// Part 1
+console.log('Part 1');
+
+var count = 0;
+var shopping = function() {
+	for (var i = 0; i < products.items.length; i++) {
+		if (products.items[i].kind === "shopping#product") {
+			count += 1;		
+		}
+	} console.log(count);
+} 
+
+shopping();
+console.log('');
```

Comment:
there's nothing wrong with encapsulating the search into function and then calling it.... but in this case where we only run it once, it doesnt really serve any purpose.

good work nonetheless